src/main/kotlin/com/openlattice/linking/blocking/ElasticsearchBlocker : Blocker.kt

```diff
@@ -75,7 +75,7 @@ class ElasticsearchBlocker(
         val authorizedPropertyTypes = propertyTypes.getAll(personEntityType.properties)
 
         val blockedEntitySetSearchResults = elasticsearch.searchEntitySets(
-                entitySets.values.filter { it.entityTypeId == personEntityType.id }.map(EntitySet::getId),
+                entitySets.values.filter { it.entityTypeId.equals(personEntityType.id) }.map(EntitySet::getId),
```

Comment:
This isn't necessary in Kotlin. The equals operator invokes .equals as appropriate.